build_pip_pkg.sh

```diff
@@ -44,6 +44,7 @@ function main() {
   cp ${PIP_FILE_PREFIX}setup.py "${TMPDIR}"
   cp ${PIP_FILE_PREFIX}MANIFEST.in "${TMPDIR}"
   cp ${PIP_FILE_PREFIX}LICENSE "${TMPDIR}"
+  touch ${TMPDIR}/stub.cc
```

Comment:
Sean, I don't find where we use this file. Could you point out the place or it is just for testing?

Reply:
Yeah good find... without that sort-of hack the whl fails manylinux2010 compliance because:
```
RuntimeError: Invalid binary wheel, found the following shared library/libraries in purelib folder: ... The wheel has to be platlib compliant in order to be repaired by auditwheel.
```

This is the same patch as is done on TF-Core:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/devtoolset/platlib.patch